### PR TITLE
Docs are wrong cache_dir (bool) and cache_file (str) cannot be passed as params + 1 bug

### PR DESCRIPTION
**Pull request by damon-atkins**
### What does this PR do?
Doco Fix for windows pkg.install

### Previous Behavior
`if pkginfo['version_num'].get('allusers', True):`

### New Behavior
`if pkginfo[version_num].get('allusers', True):`

### CODE REVIEW

**Comment by damon-atkins** [comment]
Hi @twangboy are you happy for this bit of doco to be removed.
Cheers

**Comment by damon-atkins** [comment]
Notice a bug
`if pkginfo['version_num'].get('allusers', True):`
vs
`if pkginfo[version_num].get('allusers', True):`

File: salt/modules/win_pkg.py [hunk 1]
```diff
@@ -913,18 +913,6 @@ def install(name=None, refresh=False, pkgs=None, **kwargs):
                 # Version 1.2.3 will apply to packages foo and bar
                 salt '*' pkg.install foo,bar version=1.2.3
 
-        cache_file (str):
-            A single file to copy down for use with the installer. Copied to the
-            same location as the installer. Use this over ``cache_dir`` if there
-            are many files in the directory and you only need a specific file
-            and don't want to cache additional files that may reside in the
-            installer directory. Only applies to files on ``salt://``
```
Comment (twangboy) [inline salt/modules/win_pkg.py]: Please move this to `salt/doc/topics/windows/windows-package-manager.rst` right above `cache_dir` (approx line 480)

**Review by twangboy — CHANGES_REQUESTED** [review]
I'm good with this PR pending the change I requested.

File: salt/modules/win_pkg.py (hunk 1 above)
Comment (damon-atkins) [reply salt/modules/win_pkg.py]: Done

**Review by twangboy — APPROVED** [review]

**Review by rallytime — APPROVED** [review]

**Comment by damon-atkins** [comment]
Hi @rallytime you may wish to push this some change all the way up to 2017.2 given `pkginfo['version_num']` should be  `pkginfo[version_num]`

**Pull request merged**